app/assets/stylesheets/global.scss

```diff
@@ -3,11 +3,19 @@
 body, html { height: 100%; }
 body {
     margin: 0 0 0 0;
-    font-family: Helvetica, Verdana, sans-serif;
+    font-family: FreeSans;
```

Comment:
This surprises me! Won't this change the site font? We might want to use this as a fallback, instead of making it the primary, since people had quite strong opinions on fonts in the past.

Reply:
I would guess this was a relic of running the site diff images, I certainly wouldn't plan to merge it in - @teceler for input?

Reply:
whoops that got accidentally committed, yeah